### DISCUSSION TITLE
How to disable Auto-discovery topics?

### DISCUSSION THREAD

**Discussion by bam80** [arendst/Tasmota#20398] [Q&A] [answered]
Every boot I see a bunch of `tasmota/discovery/...` topics published.
I don't need that, but didn't find a way to switch them off.
I tried to play with `SetOption19` but no luck (on reboot it resets to `0`).

Any advice?

**Comment by sfromis** [comment, marked as answer]
Only way is doing a custom build without the discovery feature. Or just ignore that little bit of harmless traffic.
